src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/services/LibraryService.kt

```diff
@@ -69,6 +71,26 @@ class LibraryService (
     return LibraryResult(null, LibraryStatus.LIBRARY_DOES_NOT_EXIST)
   }
 
+  fun addToLibraries(libraryIds: List<String>?, gameId: String): MultiLibraryResult {
+    val failedLibraries = mutableListOf<LibraryResult>()
+    if (libraryIds.isNullOrEmpty()) {
+      return MultiLibraryResult(emptyList(), MultiLibraryStatus.ALL_LIBRARIES_DO_NOT_EXIST)
+    }
+    if (!isValidGameInstanceId(gameId)) {
+      return MultiLibraryResult(emptyList(), MultiLibraryStatus.GAME_DOES_NOT_EXIST)
+    }
+    libraryIds.forEach { libraryId ->
+      val result = addToLibrary(libraryId, gameId)
+      if (result.libraryStatus != LibraryStatus.SUCCESS) {
+        failedLibraries.add(result)
+      }
+    }
+    if (failedLibraries.isNotEmpty()) {
+      return MultiLibraryResult(failedLibraries, MultiLibraryStatus.INDIVIDUAL_LIBRARY_ERROR)
+    }
+    return MultiLibraryResult(emptyList(), MultiLibraryStatus.SUCCESS)
```

Comment:
Wouldn't we want to return the libraries here, not an empty list?

Reply:
I could return the library ids here, but I don't think we should return the full libraries. Theoretically we could have like 100 libraries here and if we processed each one it would get crazy quickly. I think just returning OK should be good here.

Reply:
Makes sense, works with me!